doc/wallet-backup.md

```diff
@@ -14,7 +14,7 @@ Note that a backup is a duplicate of data needed to spend ZEC so where you keep
 
 These instructions are specific for the officially supported Zcash Linux client. For backing up with third-party wallets, please consult with user guides or support channels provided for those services.
 
-There are multiple ways to make sure you have at least one other copy of the private keys needed to spend your ZEC and view your shielded ZEC.
+There are multiple ways to make sure you have at least one other copy of the private keys needed to spend your ZEN.
```

Comment:
For uniformity, I would change all the other instances of `ZEC` to `ZEN`, and `ZCash` to `Zen`

Reply:
Done, thanks!